sdk/python/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb

```diff
@@ -14,22 +13,17 @@
     "- A Compute Cluster. [Check this notebook to create a compute cluster](../../../resources/compute/compute.ipynb)\n",
     "- A python environment\n",
     "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section\n",
-    "- The latest MLFlow packages:\n",
     "\n",
-    "    pip install azureml-mlflow\n",
-    "\n",
-    "    pip install mlflow\n",
     "\n",
     "**Learning Objectives** - By the end of this tutorial, you should be able to:\n",
     "- Connect to your AML workspace from the Python SDK\n",
     "- Create an `AutoML classification Job` with the 'classification()' factory-fuction.\n",
-    "- Submit and run the AutoML classification job\n",
+    "- Train the model using AmlCompute by submitting/running the AutoML training job\n",
     "- Obtaing the model and score predictions with it\n",
-    "- Leverage the auto generated training code and use it for retraining on an updated dataset\n",
     "\n",
     "**Motivations** - This notebook explains how to setup and run an AutoML classification job. This is one of the nine ML-tasks supported by AutoML. Other ML-tasks are 'regression', 'time-series forecasting', 'image classification', 'image object detection', 'nlp text classification', etc.\n",
```

Comment:
nit: Description says 9 ML task but only 5 are listed.  Same for other notebooks

Reply:
Just some are listed, etc., is given